renderer/src/components/Calendar/Calendar.tsx

```diff
@@ -70,13 +70,8 @@ export const Calendar = ({
       <p>
         (out of {daysRequiredHours})
         {!!overUnderHours && (
-          <span
-            className={
-              (overUnder === "overtime" && "text-green-500/50 ml-1") ||
-              (overUnder === "undertime" && "text-red-500/50 ml-1")
-            }
-          >
-            {overUnder === "undertime" && "-"}
+          <span className={`ml-1 ${overUnder === "undertime" && overUnderHours >= 28800000 && "text-red-500/50"}`}>
```

Comment:
I would recommend to use `8 * MS_PER_HOUR`(from constants) instead of `28800000`. It will be more clearly